### ISSUE TITLE
Pack binary operations

### ISSUE THREAD

**Issue by astojilj** [tensorflow/tfjs#951]
Binary mul, div, floorDiv, add and sub operations are a good candidate for packing implementation.
Following batchNorm packing (Issue #774), this is especially useful if someone uses transform_graph with options _fold_batch_norms fold_old_batch_norms_ before tensorflowjs_converter - batchNorm operations are replaced by biasAdd ops (binary add operation).
I'm preparing [pull request](https://github.com/astojilj/tfjs-core/commit/d01d6f2f86967760a0a8204ac2470898c966954b).

**Comment by rthadur** [comment]
#1412 is merged so closing this PR
